### ISSUE TITLE
Add ability to fold lists and tag (and whatever) groups in main window

### ISSUE THREAD

**Issue by aisbergde** [mpcjanssen/simpletask-android#1074]
if entries are grouped by list or tag elements (or by whatever elements) it would be fine to hide / show entries by clicking or long tapping on the group header. The number of items should still be shown, but not the entries.

This could be used for quick analysis and hiding not needed details instead of changing the filter.

this is not #918 which is about the left side filter.

**Comment by mpcjanssen** [comment]
I don't think this is doable in the standard RecyclerView. It would require a special widget or logic.
